appveyor.yml

```diff
@@ -0,0 +1 @@
+support/appveyor.yml
```

Comment:
I don't think this is necessary. `appveyor.yml` is intentionally placed in `support` to reduce clutter in the top-level directory.